index.html

```diff
@@ -4,9 +4,320 @@
y2bVt01w/73B3X61WT+yL4Yc6CKDdvPX3ccznglAIX7u9rGVqI4YZEIb56G6ej4CJDXlYAgI+5heO+AAQ9LXt+mh/k6QuB5CuNqSk2lDItZXVyBaKkLXXVrhDVOCB0RZaUoakqWikQNaBY4kPIdbOzGsULLd5LYeNMajPA0xJUUJXCaEhgpKe0hHumX5Ihmi2yEJ4Rqq5W7jt1u2+X7VN187n2haWlqUJUT2ysdmVbr6x2L602Nnp+76UIqqj41ea9yCAYiStenitlcIIJytVIxSRRl+EFtEiIuHe9ysi6mqxWu0c2TraNTMLDtUF4QazGGq4jS4/WNOvrsYmDvK0eoYjx3VoHW+d7nQEsK5HRcSJDfnM9vlLZz69deL2YbFQKWswNAg4WoIA3WqIhlWPKkXdC4UREeKwSO5gokvuBujI+ATCHEEIRvIkkDxEqEbmHlJzpMrhItYRpXtg5Pq/2EcmTfO7gAsS2yYIhll6/QERhihCOCPS9K+6Ac0THYf954+ZZWjvjX9SmZn7RMz/7Od5rfX6UO+nenZrNpu98lWvWi4Xf/Ev/KWv+PKv3KNPkvzAB97/Pd/33QcaRD7VBGBNNdVUV6zOpTt6SrRdXXrUhsXhjZC7ViBG09RhsOuy8nP35XJpZl3bUg6cDzsMw6rvn3RRe7i59OWw3IUj1vTr5Cikb7OPJuqguA8xIkPA6GETAqYiL3U9sedDg0DI2nJ53U8yM/1UAPMIuFCyv5Q91An7HbIijdvXtuuRvWoav3PPkxxjMBnI8Ejb5jFtbLQyZ4yuOxl1lmlkTkpAwh1CrkWM4aAHBVYrVekRFnRPW67kfagqCHoxKryvw2pkTQ2D1dp0s6YdpGml7VCUTduUWelms/nmYmd7sdyl9UWbpLEBImIAQQ0hzR0ebrl5jIpwqorLaE4GqBYXtKpWW7Iszi8/9fGPHTtx7PY7725n8+VQ3SEoqDCz8dvaT7la99lPRL+65jjRDQWA32rECpFrblC4xjwZa7e1vcfpixc/9653ffRjSt2YbfX9SiGNSgjce5DCEJFwoaS2dSDghIxZoxpW3Q1CUoZhVc3bbkYiaPS8NEXWw0huQ3Uvonv+eZK2X6OeObXMBpHULDtCcnDStav8OBwx0lkvAGcgKBIwROaOpvmakCEBGRNRr8SqsM/HnYC7931/E4nyhil/8KlVOoVNJKxbf31Xmq/88t89m836vr/v3vv2/ym9/KfTYAKwpppqqutZIaXVpqMq+gm/ugY9T9O2tlxepy8P6IehmrVN0zTNAU2fZpaL78++NYcZw5Lm8OwqZd2acUwhTDf2dJtJc5kUEDrW+WFpJEPxNFSnRHiesO5rWkSkbda6Ic0mUJjIWLZ+o38qE9MJoQbG9yEZQZF1s7qOSxztmmPUCqbCkZJkLVBkDCNMDAuAJAkkka60jo8EwhAw8zWeBKaHfT7HIUIHPEgE0wU6oqg4QYLegaLS1Lrq+5UNfdOvStc1Tdt2VdqGTcPSsG3bRko36xaLxe52rX1UJ021Sb2jkAG6KOn0gjCQ7kEG3Cwco7gSFCWpFDStaDnVdU278dCnP3rm4c/cfc99J049Z3e3F1GZzZquG3fw6UwC13zAucK1fcKtbs5JcOQwzuujr77ntt/62IdW3e1bJ+7wfrXSaEojqlH7Gj2lES0q9L66DQTYwNxJGR3bzNMmr18tpbQBepg4QUuwm1LWWuQ9AqaLKhAynsmJcOUQZKQmJpV6w0xhwBq+GkGnRPpiPaGlVTyZFK51OAXMbByl3MY4iieYPWutN5ef1IHqB0WeHfOZtdT82r9trfV6EuGnelZqtVr+zf/5f+Bj0xoi4G472zt9308TxwRgTTXVLQELXTVT3pij2xgFhrWP8lSfc7VNY7Vez/xpd1+uVkOtTdM0pXB9O/paQVd1cip9KnXLpRDGEwxrozBnNGxnxIhdIdKRJgClRFgdrZFBMEIAy3d1xBhEyGAkXuSJUMVaTegUyUY40avRST2FgoldZTYZRjesIBAhY/fMdUpiEiqANSeKwIidgSAjDOCYSzgSNSCMAB2kREoYM4gwW+FsbA1UolAigbAYTXEiu14hsBZRekQEKapCNi4iQoZYXa765VCHvmmGoW+atnRdaRvpZiiNzMu8a9uNWb9YLHd3+9Wi1mE0vU+veIFGyqc4aqkkwiLCJMTDAdASaqMm3UW6225/zomTtz/8mU9+5Lc/8Jznx/Ne9KpLu7vabjTdbI8tgjX9av/MdZ3nr+uMZB2C3iNAhXxOiWmPCSJc//K4JKy8Cr2uNv2hV99z8iMPPnr+M9ubJ56njQ/Wi7ZUUlXbDFzAalhSGERRVPemtECY1Vpr18y91tVqtdlthLuHEDGuVkQiqYZ0xtoNjxFuokUuCwUDQKY/pJYwWZQ5qjAcUAQzb3QtC85wQzpc8yEPx5hkmBGjTM4pgcx22DuFOOJm5j4MQ/p830QnGMmh1gO66ppSumfJA7sfhgMSefXDMAFYt3yZ2a+/49dFZGNjY+/rFsrXfd3X7+7ufPrBT0+ecROANdUE+twEi85nsKq+QSGtiIwg5Ei8n+oaVNd15n6dLSTMzMx6kaaUUkrmgj+DxWtE5A3YodZqhgm6eqr94eG5foQRkD1yQ2Q02HgCicAswkd0KhDuJBiS7laR4YJrMCgyNUzoXhNlcjop4UGGUBxpmSUYqVkAwseeMHVniLARt3VfixQJD8pobzV+Xja94Ums8gCg41fHzDOLtcIRBASEINycQWgkTObZ+jpD3ExUoER27+GjjVRkFCM8+WcBoTsiAwrZtJRSe61WbRjclnXVa9u0fdc2bTMbSjdj24aKzLpZ27azjWG1XGxvr1a77qaqlNzLZJKt0bdAqDMk4GGpcTS6S6odRRhi4VrKC17yyuOnnvPw2fMrbTduO+HhIpLuV7HfkPsGmH+vlGVdo+HoMN4tp1xjivVjAK0rjcwSOHKv3eqR++86emZ79alHP7y
```

Comment:
- [ ] We are not working in the contact section and form either but good job doing it 👏🏽 👏🏽 👏🏽 

![image](https://user-images.githubusercontent.com/58642949/187034879-9986573b-086d-4f3e-832d-c2528f237dbb.png)